README.md

```diff
@@ -79,42 +77,50 @@ In Wave 1 we will create the `Vendor` class.
 
 ### Wave 2
 
-In Wave 2 we will create the `Item` class and the `get_by_category` method.
+In Wave 2 we will create the `Item` class and the `Vendor` class' `get_by_id` method.
 
 - There is a module (file) named `item.py` inside of the `swap_meet` package (folder)
 
 - Inside this module, there is a class named `Item`
-- Each `Item` will have an attribute named `category`, which is an empty string by default
-- When we initialize an instance of `Item`, we can optionally pass in a string with the keyword argument `category`
-- Instances of `Vendor` have an instance method named `get_by_category`
-  - It takes one argument: a string, representing a category
-  - This method returns a list of `Item`s in the inventory with that category
+- Each `Item` will have an attribute named `id`, which is a unique integer by default
+  - There are many ways to generate numbers, but generating numbers without duplicates takes some care. Happily, Python has a package called `uuid` that can help!
+    - If we import the [`uuid` package](https://docs.python.org/3/library/uuid.html) in `item.py`, with a little research we can use one of the functions `uuid` provides to create large unique numbers meant to be used as identifiers
+    - This package creates `UUID` objects, its functions don't directly return an integer, **but** `UUID` instances have [an attribute `int`](https://docs.python.org/3/library/uuid.html#uuid.UUID.int) which allow us to access their value as an integer
+- When we initialize an instance of `Item`, we can optionally pass in an integer with the keyword argument `id` to manually set the `Item`'s `id`
+- Each `Item` will have a function named `get_category`, which will return a string holding the name of the class
```

Comment:
Would it be too much of a give away to add a bullet point that gives an example of how we can create an integer UUID?

ex) `example_uuid = uuid.uuid4().int`
or `example_uuid = uuid.int`

We would probably also want to explain why we use uuid4() or perhaps give both options and encourage them to research why the former may be the better option.